File: scripts/sort_key/category.py
import enum


class Category(enum.Enum):
    BUILDING_BLOCKS = 0
    COLORED_BLOCKS = enum.auto()
    NATURAL_BLOCKS = enum.auto()
    FUNCTIONAL_BLOCKS = enum.auto()
    REDSTONE_BLOCKS = enum.auto()
    TOOLS_AND_UTILITIES = enum.auto()
    COMBAT = enum.auto()
    FOOD_AND_DRINKS = enum.auto()
    INGREDIENTS = enum.auto()
    SPAWN_EGGS = enum.auto()
    OTHER = enum.auto()
    UNSORTED = enum.auto()

File: crates/material-core/src/category.rs
#[derive(
    Debug,
    Clone,
    Copy,
    PartialEq,
    Eq,
    Hash,
    serde::Serialize,
    serde::Deserialize,
    strum::Display,
    strum::FromRepr,
)]
#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
pub enum Category {
    BuildingBlocks,
    ColoredBlocks,
    NaturalBlocks,
    FunctionalBlocks,
    RedstoneBlocks,
    ToolsAndUtilities,
    Combat,
    Ingredients,
    FoodAndDrinks,
    SpawnEggs,
}
